src/tasks/checkEnvironment.js

```diff
@@ -29,14 +31,11 @@ var self = {
       return false;
     }
 
-    // check for windows or Mac OSX
-    if (['win32', 'darwin'].indexOf(process.platform) > -1) {
-      return self.boot2dockerExists();
-    }
```

Comment:
I updated the text from the error messages to reflect reality.  Also, updated links in README.